### PR TITLE
Use presentationId and componentId for getTweets

### PR DESCRIPTION
**Pull request by fjvallarino**
## Description
First step towards migrating to using `presentationId` and `componentId. This adds a new endpoint but leaves the existing one untouched, in order to not affect other tasks.

Next set of PRs will include environment handling and caching (separate PRs)

## Motivation and Context
Part of our epic.

## How Has This Been Tested?
It can be validated here: https://services-stage.risevision.com/twitter/get-presentation-tweets?presentationId=21c97752-0b8b-4a0c-852c-83c0471a3e00&componentId=rise-data-twitter-01

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

**Comment by fjvallarino** [comment]
@santiagonoguez @stulees please review

File: test/unit/timelines.tests.js [hunk 1]
```diff
@@ -536,4 +536,37 @@ describe("Timelines", () => {
...
+        assert(!res.json.called);
+
+        assert(res.status.called);
+        assert.equal(res.status.lastCall.args[0], BAD_REQUEST_ERROR);
+
+        assert(res.send.called);
+        assert.equal(res.send.lastCall.args[0], "Component id was not provided");
+      });
+    });
+  });
```
Comment (santiagonoguez) [inline test/unit/timelines.tests.js]: @fjvallarino aren't you forgetting the successful path ? or that will come later when the other endpoint is removed ?

File: src/config.js [hunk 2]
```diff
@@ -14,5 +14,7 @@ module.exports = {
   retryLoadInSeconds: 30,
   numberOfCachedTweets: 100,
   redisCacheHostname: "ts-redis-master",
-  redisOtpHostname: "otp-redis-master"
+  redisOtpHostname: "otp-redis-master",
+
+  coreBaseUrl: "https://rvacore-test.appspot.com/_ah/api"
```
[the remark this answers is not in the document]
Comment (fjvallarino) [reply src/config.js]: @santiagonoguez in a separate PR, using an environment variable

File: src/core/index.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    throw Error(resp.statusText);
+  })
+  .then(resp => {
+    if (resp.items && resp.items.length > 0) {
+      return resp.items[0];
+    }
+
+    throw new Error("Invalid response");
```
[the remark this answers is not in the document]
Comment (fjvallarino) [reply src/core/index.js]: @santiagonoguez none, I can remove `new` for consistency if we're already using it that way

File: src/core/index.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
+const config = require("../config");
+const utils = require("../utils");
+
+const validationErrorFor = message => Promise.reject(new Error(message));
```
[the remark this answers is not in the document]
Comment (fjvallarino) [reply src/core/index.js]: @santiagonoguez makes sense, I'll move it

File: test/unit/timelines.tests.js (hunk 1 above)
Comment (fjvallarino) [reply test/unit/timelines.tests.js]: @santiagonoguez no, it's intentional. The successful path is already covered by existing test cases, and this extra endpoint will take the place of the original one (meaning, I will move the new tests into the existing test suite)

**Pull request merged**